mmv1/templates/terraform/constants/cloud_run_service.go.erb

```diff
@@ -23,6 +24,16 @@ func cloudrunAnnotationDiffSuppress(k, old, new string, d *schema.ResourceData)
 	return false
 }
 
+func cloudrunTemplateAnnotationDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	// Suppress diffs for the annotations provided by API
+	if old == "gvisor" && new == "" {
```

Comment:
Can we check the key here too? We only want to suppress diffs on the `template.metadata.annotations.run.googleapis.com/sandbox` key ( not sure if that is exactly what the key ends up being)

Reply:
You can use regexp MatchString(k) like in others

Reply:
done